qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/CommentCheck/violations.txt

```diff
@@ -0,0 +1,5 @@
+6:This kind of comment is not allowed.
+9:First sentence in a comment should start with a capital letter.
```

Comment:
@volodya-lombrozo  Duplication occurs because we have this check triggered twice, once in the class com.qulice.checkstyle.MultiLineCommentCheck and the second time in the class com.qulice.checkstyle.SingleLineCommentCheck.
Unfortunately, I don't know where it would be better to place the description of this case.